src/app/EventManagement.h

```diff
@@ -566,6 +576,7 @@ class EventManagement
     Messaging::ExchangeManager * mpExchangeMgr = nullptr;
     EventManagementStates mState               = EventManagementStates::Shutdown;
     uint32_t mBytesWritten                     = 0;
+    System::Mutex mAccessLock;
```

Comment:
Does the mutex act as a noop without CHIP_SYSTEM_CONFIG_NO_LOCKING defined?

Reply:
removed, thanks